sass/_Fonts.scss

```diff
@@ -0,0 +1,6 @@
+@font-face {
+  font-family: 'Lato';
+  src: __FONTS;
+  font-weight: normal;
+  font-style: normal;
+}
```

Comment:
Becomes:
```css
@font-face {
  font-family: 'Lato';
  src: url('../fonts/lato.woff2'), url('https://static.cloud.coveo.com/searchui/v2.10091/2/fonts/lato.woff2'), url('https://staticdev.cloud.coveo.com/searchui/v2.10091/2/fonts/lato.woff2'), url('../fonts/lato.woff'), url('https://static.cloud.coveo.com/searchui/v2.10091/2/fonts/lato.woff'), url('https://staticdev.cloud.coveo.com/searchui/v2.10091/2/fonts/lato.woff');
  font-weight: normal;
  font-style: normal;
}
```

Reply:
If `../fonts/lato.woff2` is absent, this will create a 404 error every time, right ?

I think it would be better to just reference our CDN

Reply:
I'll make it CDN in prod and local in dev.

Reply:
I'll move the local CDN to the end of the list. In dev, I believe we may still need a fallback when the current package.json version isn't on the CDN.